src/RichText/Toolbar/Toolbar.tsx

```diff
@@ -35,6 +37,7 @@ export function Toolbar({
   editor,
   hidden = undefined,
   items = DEFAULT_TOOLBAR_ITEMS,
+  ListComponent = FlatList,
```

Comment:
This is a great ability regardless the bottom-sheet, can we add this to the props table on: https://10play.github.io/10tap-editor/docs/api/Components#toolbar

Reply:
done